### PR TITLE
Update HCA compensation question

### PR DESCRIPTION
**Pull request by jbalboni**
screen shot 2017-10-18 at 5 23 39 pm

### CODE REVIEW

**Comment by rlbaker** [comment]
It appears the latest test run failed due to a timing issue. Don't know if this is something that will be fixed by just re-running the test suite or not.

**Comment by jbalboni** [comment]
@rlbaker Looks like I missed some tests (plus there was some other failure that I merged master to fix).

**Review by rlbaker — APPROVED** [review]
Looks good to me. This probably shouldn't be merged before department-of-veterans-affairs/vets-api/pull/1465 is merged and deployed though. I can update here once it's ready to go.

**Comment by rlbaker** [comment]
https://github.com/department-of-veterans-affairs/vets-api/pull/1465 has been merged and should go out on todays deploy. This can safely be merged.

File: src/js/hca/config/migrations.js [hunk 1]
```diff
@@ -62,5 +62,56 @@ export default [
...
+    if (compensableVaServiceConnected === false && isVaServiceConnected === false && receivesVaPension === true) {
+      return {
+        formData: _.set('vaCompensationType', 'pension', newFormData),
+        metadata
+      };
+    }
+
+    // More than one option was chosen, or not all were filled out, so go back to the page and make the user pick again,
+    // because we don't know for sure what they meant to pick
+    return {
```
Comment (cehsu) [inline src/js/hca/config/migrations.js]: I wonder if this would now be a good opportunity for a tooltip?

**Review by cehsu — APPROVED** [review]
LGTM. Had a question about whether it might make sense to potentially create a tooltip to help explain this question. Should `yarn.lock` be included?

File: src/js/hca/config/migrations.js (hunk 1 above)
Comment (jbalboni) [reply src/js/hca/config/migrations.js]: Could be, if @peggygannon thought that was useful.

**Comment by peggygannon** [comment]
@cehsu I am just seeing this screenshot. We need to add a ' mark in the work don't. 

it should be don't (not dont). 

and yes, this could be a potential one for expandable text.

**Comment by jbalboni** [comment]
@peggygannon Ack, good catch, I forgot to go back and add the fancy apostrophe.

**Pull request merged**